mex/extractors/endnote/main.py

```diff
@@ -56,8 +56,8 @@ def endnote_extracted_consents(
     return endnote_extracted_consents
 
 
-@asset(group_name="endnote")
-def endnote_extracted_bibliographic_resources(
+@asset(group_name="endnote", metadata={"entity_type": "bibliographic-resource"})
+def extracted_endnote_bibliographic_resources(
```

Comment:
```suggestion
def endnote_extracted_bibliographic_resources(
```
I think the wording pattern is: extractor_itemtype, so first "endnote" then "extractedBibliographicResource" was correct.